src/main/java/frc/robot/parameters/AprilTagFieldParameters.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2025 Newport Robotics Group. All Rights Reserved.
+ *
+ * Open Source Software; you can modify and/or share it under the terms of
+ * the license file in the root directory of this project.
+ */
+ 
+package frc.robot.parameters;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+
+public enum AprilTagFieldParameters {
+  k2025ReefscapeWelded(AprilTagFields.k2025ReefscapeWelded),
+  k2025ReefscapeAndyMark(AprilTagFields.k2025ReefscapeAndyMark);
```

Comment:
The enum names are displayed in the chooser so we should omit the `k2025`.

```suggestion
  ReefscapeWelded(AprilTagFields.k2025ReefscapeWelded),
  ReefscapeAndyMark(AprilTagFields.k2025ReefscapeAndyMark);
```